### ISSUE TITLE
Add Annotator-function to directly enter (download-)link to manuscript

### ISSUE THREAD

**Issue by LukasRoeseler** [forrtproject/FReD#46]
If I want to check references from a pre-print, it is tedious to download or copy them first. I would like to instead simply paste the pre-print link (https://osf.io/preprints/psyarxiv/abcde) or the corresponding download-link (https://osf.io/abcde/download/) somewhere in the annotator and have it download the manuscript directly. @LukasWallrich do you think that this can easily be implemented?

**Comment by LukasWallrich** [comment]
Yes, for sure - many R functions can directly accept URLs in the file
argument. However, many links are not direct download links but links to a
download page - there we might run into trouble. I will have a look when
UKRI is in.
